src/zenml/cli/text_utils.py

```diff
@@ -0,0 +1,74 @@
+#  Copyright (c) ZenML GmbH 2022. All Rights Reserved.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at:
+#
+#       https://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+#  or implied. See the License for the specific language governing
+#  permissions and limitations under the License.
+from typing import List
+
+from rich.markdown import Markdown
+
+zenml_go_welcome_message = Markdown(
+    """
+# ⛩ Welcome to ZenML!
+"""
+)
+
+zenml_go_email_prompt = Markdown(
+    """
+Here at ZenML we are working hard to produce the best
+possible MLOps tool. In order to solve real world problems
+we want to ask you, the user for feedback and ideas. If
+you are interested in helping us shape the MLOps world
+please leave your email below (leave blank to skip). We will
+only use this for the purpose of reaching out to you for a
```

Comment:
We will also use it to better understand usage so maybe this is incorrect

Reply:
Will expand it:

"We will
only use this for the purpose of reaching out to you for a
user interview and to better understand usage."